src/runner.ts

```diff
@@ -40,15 +69,15 @@ export class IBMiTestRunner {
             if (!compiledTestFile) {
                 IBMiTestRunner.updateTestRunStatus(run, 'testFile', { item: testFileItem });
 
-                if (testFileData.didCompile) {
+                if (testFileData.isCompiled) {
```

Comment:
I think it is important to only compile if the object is older than the source, because there may be a lot of unit tests eventually, and this should scale